src/lib/services/gauging-station-service.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const gaugingStationRepo = require('../connectors/repos/gauging-stations');
+const gaugingStationMapper = require('../mappers/gauging-station');
+const { NotFoundError } = require('../errors');
+
+const getGaugingStations = async gaugingStationId => {
+  const gaugingStationResponse = await gaugingStationRepo.findOne(gaugingStationId);
+  if (gaugingStationResponse) {
+    return gaugingStationMapper.dbToModel(gaugingStationResponse);
+  }
+  throw new NotFoundError('GaugingStations by id not found');
+};
+
+const getGaugingStationsByRef = async stationRef => {
+  const gaugingStationResponseRef = await gaugingStationRepo.findOneByStationRef(stationRef);
+  if (gaugingStationResponseRef) {
+    return gaugingStationMapper.dbToModel(gaugingStationResponseRef);
+  }
+  throw new NotFoundError('GaugingStations by ref not found');
+};
+
+const getGaugingStationConditionsForId = async gaugingStationId => {
```

Comment:
This function will hopefully become `getGaugingStationConditionsById` instead of `getGaugingStationConditionsForId` :)